examples/survey-app/src/app/pages/login/login.page.ts

```diff
@@ -55,11 +55,17 @@ export class LoginPage implements OnInit {
       this.email = '';
       this.password = '';
       this.errorMessage = '';
-      await Keyboard.hide();
+
+      if (Capacitor.getPlatform() !== 'web') {
+        await Keyboard.hide();
+      }
       this.router.navigate(['/surveys']);
     } catch (error) {
       this.errorMessage = 'Invalid login credentials';
-      await Keyboard.hide();
+
+      if (Capacitor.getPlatform() !== 'web') {
+        await Keyboard.hide();
+      }
```

Comment:
@Alex-Smith-Arcweb is this solving a bug we hadn't reported?

Reply:
Correct, sorry I meant to put this in the description.  If the keyboard hide is called on web it errors out